src/avio.jl

```diff
@@ -631,9 +631,13 @@ if have_avdevice()
         camera_devices = String[]
 
         read_vid_devs = false
-        out, err = readall_stdout_stderr(`$ffmpeg -list_devices true -f $idev -i $idev_name`)
-        buf = length(out) > 0 ? out : err
-        for line in eachline(IOBuffer(buf))
+        stdout_buf = IOBuffer()
+        stderr_buf = IOBuffer()
+        run(pipeline(ignorestatus(`$ffmpeg -list_devices true -f $idev -i $idev_name`), stdout=stdout_buf, stderr=stderr_buf))
```

Comment:
erroring here with `no method matching rawhandle(::Base.GenericIOBuffer{Array{UInt8,1}})`

Reply:
This works on windows on 1.1.0 (but not 1.0.3).
I hadn't appreciated it was a new change